portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lorna Armstrong's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Welcome to the porfolio of Lorna Armstrong.</p>
+
+      <h3>About Me</h3>
+      <p>My name is Lorna Armstrong and I am from Middlesbrough, England. 
+          I am a Computer Science student at the University of Edinburgh, 
+          heading into my third year in September 2020. Currently, I am 
+          working as a STEP Intern for Google, working remotely from Edinburgh.
+          This portfolio will show you a little bit about me.</p>
+
+      <img src= "/images/profile.jpg" alt="Profile Picture" style="width:200px;height:200px;">    
+      <img src= "/images/edinburgh.jpg" alt="Skyline of Edinburgh" style="width:400px;height:250px;">
+      <img src= "/images/skye.jpg" alt="My Border Collie, Skye" style="width:300px;height=300px;">
+      
+      <!-- Code from Original Template -->
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
```

Comment:
Nit: As you don't have a blank line at the beginning of this block, this seems a bit unnecessary. In general we tend to avoid whitespace/blank lines in HTML ([ref](https://www.w3schools.com/html/html5_syntax.asp), please search with _Blank Lines and Indentation_).

Reply:
Thanks for the feedback and the reference link!